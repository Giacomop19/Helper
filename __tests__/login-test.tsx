import * as React from 'react';
import renderer from 'react-test-renderer';

import Login from "@/app/login";

it(`renders correctly`, () => {
    const log = renderer.create(<Login></Login>).toJSON()
    expect(log).toMatchSnapshot()
})