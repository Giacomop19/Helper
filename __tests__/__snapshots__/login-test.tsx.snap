// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#11181C",
        },
        {
          "fontSize": 16,
          "lineHeight": 24,
        },
        undefined,
        undefined,
        undefined,
        undefined,
        {
          "fontSize": 20,
          "fontWeight": "bold",
        },
      ]
    }
  >
    Welcome to Helper!
  </Text>
  <Text
    style={
      [
        {
          "color": "#11181C",
        },
        {
          "fontSize": 16,
          "lineHeight": 24,
        },
        undefined,
        undefined,
        undefined,
        undefined,
        {
          "fontSize": 18,
          "margin": 24,
          "textAlign": "center",
        },
      ]
    }
  >
    Login to access your application
  </Text>
  <View
    style={
      [
        {
          "backgroundColor": "#eee",
        },
        {
          "height": 1,
          "marginVertical": 30,
          "width": "80%",
        },
      ]
    }
  />
  <TextInput
    onChange={[Function]}
    placeholder="Username"
    style={
      {
        "borderColor": "grey",
        "borderRadius": 10,
        "borderWidth": 1,
        "color": "white",
        "margin": 10,
        "padding": 10,
        "width": "80%",
      }
    }
  />
  <TextInput
    onChange={[Function]}
    placeholder="Password"
    secureTextEntry={true}
    style={
      {
        "borderColor": "grey",
        "borderRadius": 10,
        "borderWidth": 1,
        "color": "white",
        "margin": 10,
        "padding": 10,
        "width": "80%",
      }
    }
  />
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "backgroundColor": "#4287f5",
        "borderRadius": 15,
        "padding": 10,
        "width": "60%",
      }
    }
  >
    <Text
      style={
        [
          {
            "color": "#11181C",
          },
          {
            "fontSize": 16,
            "lineHeight": 24,
          },
          undefined,
          undefined,
          undefined,
          undefined,
          {
            "color": "white",
            "fontSize": 16,
            "fontWeight": "bold",
            "textAlign": "center",
          },
        ]
      }
    >
      Login
    </Text>
  </View>
  <Text
    style={
      [
        {
          "color": "#11181C",
        },
        {
          "fontSize": 16,
          "lineHeight": 24,
        },
        undefined,
        undefined,
        undefined,
        undefined,
        {
          "color": "white",
          "fontSize": 16,
          "paddingTop": 10,
          "textAlign": "center",
        },
      ]
    }
  >
    Don't have an account?
  </Text>
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
  >
    <Text
      style={
        [
          {
            "color": "#11181C",
          },
          {
            "fontSize": 16,
            "lineHeight": 24,
          },
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
        ]
      }
    >
      Sign up
    </Text>
  </View>
</View>
`;
